site/_docs/history.md

```diff
@@ -41,7 +40,7 @@ y (release manager),
 z.
 
 #### Breaking Changes
-{: #breaking-1-35-0}
+{: #breaking-1-36-0}
 
 The way of Locale parsing changed within [<a href="https://issues.apache.org/jira/browse/CALCITE-5567">CALCITE-5567</a>]
```

Comment:
If I am not mistaken, I think the intention was for these two lines to be part of the breaking-1-35 section.
However, I have the impression that the link is incorrect and should rather be CALCITE-5746, @snuyanzin could you please confirm?

Reply:
@rubenada Hi, I have confirmed. This is for 1-35.0 and The Jira ISSUE should be CALCITE-5746. Thanks for the review.

Reply:
Done.

Reply:
Thanks. Please be careful, if I am not mistaken, in the current status of the PR, the link target has one ticket number, but the link text has a different ticket number, could you please double-check?

Reply:
Done. Thanks very much.